protocol-spec/generic.md

```diff
@@ -134,5 +144,148 @@
 ]
 ```
 
+## VibrateCmd
+
+**Description:** Causes a toy that supports vibration to run specific vibration motors at a certain speeds. Devices with multiple vibrator features may take multiple values. The [FeatureCount](enumeration.md#messageattributes) attribute for the message in the [DeviceList](enumeration.md#devicelist)/[DeviceAdded]enumeration.md#deviceadded) message will contain that information.
```

Comment:
Here and in other generic message descriptions, links for DeviceAdded are missing an opening paren.